lib/controllers/CMakeLists.txt

```diff
@@ -10,13 +10,7 @@ target_link_libraries(ActuatorController
         )
 target_set_mbed_dependency(ActuatorController)
 
-add_library(Motor STATIC)
-target_sources(Motor PRIVATE src/Motor.cpp)
-target_include_directories(Motor PUBLIC include)
-target_set_mbed_dependency(Motor)
-
-add_library(ServoMotor STATIC)
-target_sources(ServoMotor PRIVATE src/ServoMotor.cpp)
-target_include_directories(ServoMotor PUBLIC include)
-target_link_libraries(ServoMotor PRIVATE ContServo Motor)
-target_set_mbed_dependency(ServoMotor)
+# add_library(Motor STATIC)
+# target_sources(Motor PRIVATE src/Motor.cpp)
+# target_include_directories(Motor PUBLIC include)
+# target_set_mbed_dependency(Motor)
```

Comment:
I commented it out cuz it was breaking the compilation, since the current ActuatorController is still based off old code. This file should get updated once ActuatorController gets updated

Reply:
ok. i'd leave a todo with your username so you don't forget lol